iothub/service/src/Device.cs

```diff
@@ -109,9 +110,29 @@ public Device(string id)
         public virtual DeviceCapabilities Capabilities { get; set; }
 
         /// <summary>
-        /// Scope to which this device instance belongs to
+        /// The scope of the device. For edge devices, this is auto-generated and immutable. For leaf devices, set this to create child/parent
+        /// relationship.
         /// </summary>
+        /// <remarks>
+        /// For edge devices, the auto-generated device scope consists of a prefix "ms-azure-iot-edge://", the name of the parent edge device, a dash, and the
+        /// generation Id of the parent edge device (e.g. ms-azure-iot-edge://MyEdgeDevice-000000000000000000).
+        ///
+        /// For leaf devices, the value to set a parent edge device can be retrieved from the parent edge device's <see cref="Scope"/> property.
+        /// </remarks>
         [JsonProperty(PropertyName = "deviceScope", NullValueHandling = NullValueHandling.Ignore)]
         public virtual string Scope { get; set; }
+
+        /// <summary>
+        /// The scopes of the upper level edge devices if applicable.
+        /// </summary>
+        /// <remarks>
+        /// For edge devices, the value to set a parent edge device can be retrieved from the parent edge device's <see cref="Scope"/> property.
+        ///
+        /// For leaf devices, this could be set to the same value as <see cref="Scope"/> or left for the service to copy over.
```

Comment:
Is it true that the service will copy it over? I remember having a discussion on teams talking about how this was not the case.